src/dx/access/builders/mod.rs

```diff
@@ -19,15 +19,15 @@ pub use revoke::RevokeTokenRequestWithDeserializerBuilder;
 pub use revoke::{RevokeTokenRequest, RevokeTokenRequestBuilder};
 pub mod revoke;
 
+use crate::dx::pubnub_client::PubNubClientInstance;
```

Comment:
Isn't `pubnub_client` is redundant in path? I can see that `PubNubClientInstance` is re-exported in `lib.rs`.

Reply:
I'm gonna check that tomorrow's morning. ;o

Reply:
It's not re-exported. ;o

Reply:
Oh, indeed. Should we? :)

Reply:
Imo not. Developers should use it without PubNubMiddleware so we shouldn't reexport that. ;d